pkg/plugins/layout.go

```diff
@@ -535,6 +546,10 @@ func (gd GdiskCall) buildOptions() []string {
 		opts = append(opts, "-P")
 	}
 
+	if gd.forceGPT {
```

Comment:
if this is run in a rpi3 this will break as it transforms the mbr partition into gpt and will not be bootable anymore :/

Reply:
argggg ... good point, thanks!